### PR TITLE
HZN-732: Add smoke tests for the Topology UI

### PR DESCRIPTION
**Pull request by j-white**
JIRA: http://issues.opennms.org/browse/HZN-732
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS679

### CODE REVIEW

File: smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java [hunk 1]
```diff
@@ -61,59 +62,61 @@
     /**
      * Class to control the inputs of the "Business Service Edit"-Window
      */
-    private class BsmAdminPageEditWindow {
+    public static class BsmAdminPageEditWindow {
```
Comment (mvrueden) [inline smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java]: Can we make the classes not inner classes and move them to the src package instead?

File: smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+        @Override
+        public String toString() {
+            return String.format("VisibleVertex[label='%s']", label);
+        }
+    }
+
+    /**
+     * Controls the workflow of the "Topology UI" Page
+     */
+    public static class TopologyUiPage {
```
Comment (mvrueden) [inline smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java]: Can we name this `TopologyUIPage` to be consistend with the `TopologyUI` class?

File: smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+                                + " Unable to determine if the item is checked or unchecked.");
+                }
+            } else {
+                throw new RuntimeException("Element has no CSS classes!"
+                        + " Unable to determine if the item is checked or unchecked.");
+            }
+        }
+    }
+
+    public static class TopologyUiSearchResults {
```
Comment (mvrueden) [inline smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java]: Can we rename this to `TopologyUISearchResults` just to be consistend with the `TopologyUI` class?

File: smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java (hunk 1 above)
Comment (j-white) [inline smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java]: Most of the method they need access to are *protected*, so we need to keep them in the same package. That said, I'd rather keep them grouped together here than breaking them out in *org.opennms.smoketest*.

**Comment by j-white** [comment]
I'm going to retarget this to the features/atlas branch to make sure we address any possible intermittent failures before it's merged back to develop.

**Pull request closed without merging**